### PR TITLE
Adapt a new structure of module arguments 

### PR DESCRIPTION
**Pull request by revit13**
This PR implements part of issue #56 
It replaces the "copy" with "assets" in the batchTranfer/StreamTransfer resources.

Handling the app-uuid will be done in a separate issue dedicated to improve logging.

### CODE REVIEW

**Review by shlomitk1 — COMMENTED** [review]
I believe you need assets[0]

**Comment by revit13** [comment]
@shlomitk1 no, it does not work with assets[0]... `copy` and `assets` are the same.. only the name was changed.

**Comment by shlomitk1** [comment]
> @shlomitk1 no, it does not work with assets[0]... `copy` and `assets` are the same.. only the name was changed.

I don't understand. Assets is an array, copy is a single structure.

File: modules/fybrik-implicit-copy-batch/values.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ imagePullPolicy: null
 noFinalizer: "true"
 
 # copies from source
-copy:
+assets:
   source:
```
Comment (shlomitk1) [inline modules/fybrik-implicit-copy-batch/values.yaml]: assets is an array. Should be 
assets:
  ` - source`: ...

**Comment by revit13** [comment]
@shlomitk1 I fixed the argument structure. Please review again. Thanks

**Review by shlomitk1 — APPROVED** [review]

**Pull request merged**